resume-buddy/src/main/java/com/google/sps/servlets/RevieweeDataServlet.java

```diff
@@ -40,14 +40,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(revieweeEntity);
 
-    /* TODO: add a response.sendRedirect() to wherever the user should
-     * be navigated to next (either homepage or resume review page)
-     */ 
+    response.sendRedirect("resume-review.html");
   }
 
   /**
    * @return the request parameter, or the default value if the parameter was not specified by the
-   * client
+   *     client
```

Comment:
Nit: auto format must have added this extra space again before client... strange